### PR TITLE
Add via_web loki logger

### PR DESCRIPTION
**Pull request by RobertLucey**

### CODE REVIEW

**Review by conor-f — CHANGES_REQUESTED** [review]
Nice :) A few changes but should be good to go after that.

Any way you tested this locally? I guess the only changes on the server would be to update the env file?

File: src/via_web/log.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+
+logger = logging.Logger('via', logging.DEBUG)
+
+os.makedirs(
+    os.path.dirname(LOG_LOCATION),
+    exist_ok=True
+)
+
+if os.getenv('LOKI_ENDPOINT') == 'True':
```
[the remark this answers is not in the document]
Comment (RobertLucey) [reply src/via_web/log.py]: Oops, means it's broken on via-api too. Will bump that and update here

**Comment by RobertLucey** [comment]
Haven't tested, don't have a loki instance but should be straightforward enough if you update the env
Should get logs for via_web and via_api then
via-api 0.0.55 is building now, should be up in 5 mins or so

**Review by conor-f — APPROVED** [review]

**Comment by conor-f** [comment]
Added the env vars there so will see when this deploys :) Merge whenever suits

**Pull request merged**